### PR TITLE
Translate README.md

### PR DESCRIPTION
**Pull request by gcharest**
# Summary | Résumé

Proposed translation for the README.md fixing #47 

Also fixed markdown lint issues and adjusted formatting for bilingual content (Single h1)

---

# Test instructions | Instructions pour tester la modification

Open the README.md file and confirm all content has been properly translated.

---

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet

- Are there any additional updates required to the current content (EN/FR)?
- Could the striked out sections be removed? (History is preserved in git anyway so... :shrug: )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,3 +54,58 @@ resource "aws_route53_record" "ebrief-cds-snc-ca-A" {
...
+
+- _*.cds-snc.ca_
+- _*cdssandbox.xyz_
+- _*.alpha.canada.ca_.
+
+### Pourquoi en aurais-je besoin?
+
+Vous voulez une jolie URL comme `notification.alpha.canada.ca` pour votre nouveau projet ?  C'est comme ça qu'il faut faire.
+
+### Faire des requêtes DNS
```
Comment (AntoineAugusti) [inline README.md:74]:
```suggestion
### Ajouter des enregistrements DNS
```

File: README.md [hunk 2]
```diff
@@ -45,3 +54,58 @@ resource "aws_route53_record" "ebrief-cds-snc-ca-A" {
...
+
+Pour demander un nouveau sous-domaine, créez une entrée de terraforme `aws_route53_record` et soumettez un [enjeu (issue)](https://github.com/cds-snc/dns/issues/new) ~~pull request~~.
```
Comment (AntoineAugusti) [inline README.md:76]:
```suggestion
Pour demander un nouveau sous-domaine, créez une entrée Terraform `aws_route53_record` et soumettez un [enjeu (issue)](https://github.com/cds-snc/dns/issues/new).
```

Issues are not PRs

File: README.md [hunk 3]
```diff
@@ -45,3 +54,58 @@ resource "aws_route53_record" "ebrief-cds-snc-ca-A" {
...
+Un membre de l'équipe de la plate-forme examinera la demande et l'approuvera ou reviendra avec des questions supplémentaires.
+
+Nous cherchons à faciliter les choses, mais si quelqu'un veut nous aider, veuillez contacter john.obrien@cds-snc.ca
+
```
Comment (AntoineAugusti) [inline README.md:80]:
```suggestion
```

Our dear @obrien-j is leaving CDS soon so removing this sentence

File: README.md [hunk 4]
```diff
@@ -45,3 +54,58 @@ resource "aws_route53_record" "ebrief-cds-snc-ca-A" {
...
+### Exemples
+
+Sous-domaine avec un enregistrement CNAME dirigeant le trafic vers un point de terminaison Cloudfront :
+
+```json
```
Comment (AntoineAugusti) [inline README.md:85]:
```suggestion
```hcl
```

File: README.md [hunk 5]
```diff
@@ -45,3 +54,58 @@ resource "aws_route53_record" "ebrief-cds-snc-ca-A" {
...
+        "d31g0redbbnj2z.cloudfront.net"
+    ]
+    ttl     = "300"
+
+}
+```
+
+Sous-domaine avec un enregistrement A dirigeant le trafic vers une adresse IP :
+
+```json
```
Comment (AntoineAugusti) [inline README.md:100]:
```suggestion
```hcl
```

File: README.md [hunk 6]
```diff
@@ -45,3 +54,58 @@ resource "aws_route53_record" "ebrief-cds-snc-ca-A" {
...
+### Faire des requêtes DNS
+
+Pour demander un nouveau sous-domaine, créez une entrée de terraforme `aws_route53_record` et soumettez un [enjeu (issue)](https://github.com/cds-snc/dns/issues/new) ~~pull request~~.
+Un membre de l'équipe de la plate-forme examinera la demande et l'approuvera ou reviendra avec des questions supplémentaires.
+
+Nous cherchons à faciliter les choses, mais si quelqu'un veut nous aider, veuillez contacter john.obrien@cds-snc.ca
+
+### Exemples
+
+Sous-domaine avec un enregistrement CNAME dirigeant le trafic vers un point de terminaison Cloudfront :
```
Comment (AntoineAugusti) [inline README.md:83]:
```suggestion
Sous-domaine avec un enregistrement `CNAME` dirigeant le trafic vers un point de terminaison CloudFront :
```

File: README.md [hunk 7]
```diff
@@ -45,3 +54,58 @@ resource "aws_route53_record" "ebrief-cds-snc-ca-A" {
...
+    type    = "CNAME"
+    records = [
+        "d31g0redbbnj2z.cloudfront.net"
+    ]
+    ttl     = "300"
+
+}
+```
+
+Sous-domaine avec un enregistrement A dirigeant le trafic vers une adresse IP :
```
Comment (AntoineAugusti) [inline README.md:98]:
```suggestion
Sous-domaine avec un enregistrement `A` dirigeant le trafic vers une adresse IP :
```

File: README.md [hunk 8]
```diff
@@ -45,3 +54,58 @@ resource "aws_route53_record" "ebrief-cds-snc-ca-A" {
 }
 ```
 
+---
+
+## SNC DNS en tant que service
+
+Avec nos remerciements à nos amis de [18f](https://github.com/18F) pour l'[inspiration](https://github.com/18F/dns), ce dépôt est une collection de scripts Terraform pour la configuration des DNS pour les domaines gérés par SNC.
```
Comment (AntoineAugusti) [inline README.md:61]:
```suggestion
Avec nos remerciements à nos amis de [18f](https://github.com/18F) pour l'[inspiration](https://github.com/18F/dns), ce dépôt est une collection de scripts Terraform pour la configuration des DNS pour les domaines gérés par le SNC.
```

File: README.md [hunk 9]
```diff
@@ -45,3 +54,58 @@ resource "aws_route53_record" "ebrief-cds-snc-ca-A" {
...
+
+~~Actuellement, il se limite à la gestion des sous-domaines de *.cds-snc.ca.  À l'avenir, nous étendrons cette liste pour inclure le domaine cdssandbox.xyz.~~
+Pour le moment, nous gérons les domaines:
```
Comment (AntoineAugusti) [inline README.md:64]:
```suggestion
Pour le moment, nous gérons les domaines:
```

File: README.md [hunk 10]
```diff
@@ -32,7 +40,8 @@ resource "aws_route53_record" "rescheduler-cds-snc-ca-CNAME" {
 ```
 
 Subdomain with an A record directing traffic to an IP address:
-```
+
+```json
```
Comment (AntoineAugusti) [inline README.md:44]:
```suggestion
```hcl
```

File: README.md [hunk 11]
```diff
@@ -1,24 +1,32 @@
...
 
 We are investigating how to make this easier, but if anyone wants to help out, please reach out to john.obrien@cds-snc.ca
 
-## Examples
+### Examples
 
 Subdomain with a CNAME record directing traffic to a Cloudfront endpoint:
-```
+
+```json
```
Comment (AntoineAugusti) [inline README.md:29]:
```suggestion
```hcl
```

File: README.md [hunk 12]
```diff
@@ -1,24 +1,32 @@
...
+
+### Why would I need this?
 
 Do you want a pretty URL like `notification.alpha.canada.ca` for your cool new project?  This is how you do that.
 
-## Making DNS requests
+### Making DNS requests
 
-To request a new subdomain, create a `aws_route53_record` terraform entry and submit an ~~pull request~~ issue.  A member of the Platform team will review the request and either approve it, or circle back with additional questions.
+To request a new subdomain, create a `aws_route53_record` terraform entry and submit an ~~pull request~~ [issue](https://github.com/cds-snc/dns/issues/new).  A member of the Platform team will review the request and either approve it, or circle back with additional questions.
```
Comment (AntoineAugusti) [inline README.md:21]:
```suggestion
To request a new subdomain, create a `aws_route53_record` terraform entry and submit an [issue](https://github.com/cds-snc/dns/issues/new).  A member of the Platform team will review the request and either approve it, or circle back with additional questions.
```

File: README.md [hunk 13]
```diff
@@ -1,24 +1,32 @@
-# CDS DNS as a Service
+# alpha.canada.ca et al
+
+## CDS DNS as a Service
 
 With thanks to our friends at [18f](https://github.com/18F) for the [inspiration](https://github.com/18F/dns), this repository is a collection of Terraform scripts for configuring DNS for domains managed by CDS.
 
 ~~Currently this is limited to managing subdomains of *.cds-snc.ca.  In future, we will expand to include the cdssandbox.xyz domain.~~
```
Comment (AntoineAugusti) [inline README.md:7]: We can remove this sentence as you suggested

**Review by AntoineAugusti — CHANGES_REQUESTED** [review]
Salut @gcharest,

Thanks for opening this PR and sorry we took so much time to review it. Suggested a few changes for you to consider before approving.

Merci pour l'aide!

File: README.md (hunk 3 above)
Comment (gcharest) [reply README.md:79]: Yes, just learned! Sad to see him go!

**Comment by gcharest** [comment]
@AntoineAugusti Accepted all recommended changes! Thanks for getting back to me :smile:

**Review by AntoineAugusti — APPROVED** [review]
Thanks @gcharest!

**Comment by AntoineAugusti** [comment]
@gcharest

Protip, you can batch approve changes on GitHub using "Add suggestion to batch" https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request

Will save you a few clicks next time.

**Comment by gcharest** [comment]
@AntoineAugusti  Haha, thanks! Hadn't used the GitHub interface for those before :laughing: 

Continuous learning!

**Comment by gcharest** [comment]
Content merged in #183

**Pull request closed without merging**